Tests/ViewHelpers/CarsViewHelperTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+require_once 'src/ViewHelpers/CarsViewHelper.php';
+require_once 'src/Entities/Car.php';
+use PHPUnit\Framework\TestCase;
+
+class CarsViewHelperTest extends TestCase
+{
+    public function test_displayAllCars_success(): void
```

Comment:
You should have a failure test in here - what happens if you pass in an empty array of cars - you should get the message from the view helper